doc/freeze/compress.md

```diff
@@ -64,10 +64,25 @@ In the compressed DP model, the number of neighbors is precisely indexed at the
 
 Once the frozen model is obtained from DeePMD-kit, we can get the neural network structure and its parameters (weights, biases, etc.) from the trained model, and compress it in the following way:
 
+::::{tab-set}
+
+:::{tab-item} TensorFlow {{ tensorflow_icon }}
+
 ```bash
 dp compress -i graph.pb -o graph-compress.pb
 ```
 
+:::
+
+:::{tab-item} PyTorch {{ pytorch_icon }}
+
+```bash
+dp compress -i model.pt -o model-compress.pt
```

Comment:
We use `pth` to represent the final model that can be run in C++.

https://github.com/deepmodeling/deepmd-kit/blob/8355947ef052fd3806a361e25bbcff18f3e2f627/doc/backend.md?plain=1#L20-L24

```suggestion
dp compress -i model.pth -o model-compress.pth
```